### DISCUSSION TITLE
1.4.1-beta7

### DISCUSSION THREAD

**Discussion by ilyhalight** [ilyhalight/voice-over-translation#308] [Обновления]
⚠ Это бета версия и она может работать нестабильно. Обо всех проблемах рекомендуется сразу же сообщать в [Issues](https://github.com/ilyhalight/voice-over-translation/issues)

## Список изменений
- Фикс [#285](https://github.com/ilyhalight/voice-over-translation/issues/285), [#283](https://github.com/ilyhalight/voice-over-translation/issues/283), [#282](https://github.com/ilyhalight/voice-over-translation/issues/282) ([#297](https://github.com/ilyhalight/voice-over-translation/pull/297))
- Добавлено 39 новых языков перевода меню ([#297](https://github.com/ilyhalight/voice-over-translation/pull/297))
- Добавлено 2 новых языка устанавливаемых как язык видео (Латышский и Литовский - [#297](https://github.com/ilyhalight/voice-over-translation/pull/297))
- Добавлено 38 новых языков для TTS (`az`, `sq`, `am`, `hy`, `af`, `eu`, `my`, `bg`, `bs`, `cy`, `hu`, `gl`, `el`, `zu`, `kn`, `ca`, `km`, `lo`, `mk`, `ml`, `mt`, `mn`, `ne`, `nl`, `pa`, `ro`, `sr`, `si`, `sk`, `sl`, `sw`, `su`, `hr`, `et`, `da`, `fi`, `uz`, `pl`, `sv` - [#297](https://github.com/ilyhalight/voice-over-translation/pull/297))
- Улучшено форматирование ссылок на YouTube, теперь, язык должен определяться точнее ([#297](https://github.com/ilyhalight/voice-over-translation/pull/297))
- Добавлена поддержка нового встраиваемого плеера Bilibili (player.bilibili.com)
- vot-cli был перенесен в отдельный **[репозиторий](https://github.com/FOSWLY/vot-cli)**
- Удалена папка с web-server (уже давно не используется)

## What's Changed (autogenerated)
* Улучшения форматирования текста для определения языка by @SashaXser in https://github.com/ilyhalight/voice-over-translation/pull/297

**Full Changelog**: https://github.com/ilyhalight/voice-over-translation/compare/1.4.1-beta6...1.4.1-beta7

<hr /><em>This discussion was created from the release <a href='https://github.com/ilyhalight/voice-over-translation/releases/tag/1.4.1-beta7'>1.4.1-beta7</a>.</em>

**Comment by ilyhalight** [comment]
> ⚠ Это бета версия и она может работать нестабильно. Обо всех проблемах рекомендуется сразу же сообщать в [Issues](https://github.com/ilyhalight/voice-over-translation/issues)
> ## Список изменений
> 
>     * Фикс [#285](https://github.com/ilyhalight/voice-over-translation/issues/285), [#283](https://github.com/ilyhalight/voice-over-translation/issues/283), [#282](https://github.com/ilyhalight/voice-over-translation/issues/282) ([#297](https://github.com/ilyhalight/voice-over-translation/pull/297))
> 
>     * Добавлено 39 новых языков перевода меню ([#297](https://github.com/ilyhalight/voice-over-translation/pull/297))
> 
>     * Добавлено 2 новых языка устанавливаемых как язык видео (Латышский и Литовский - [#297](https://github.com/ilyhalight/voice-over-translation/pull/297))
> 
>     * Добавлено 38 новых языков для TTS (`az`, `sq`, `am`, `hy`, `af`, `eu`, `my`, `bg`, `bs`, `cy`, `hu`, `gl`, `el`, `zu`, `kn`, `ca`, `km`, `lo`, `mk`, `ml`, `mt`, `mn`, `ne`, `nl`, `pa`, `ro`, `sr`, `si`, `sk`, `sl`, `sw`, `su`, `hr`, `et`, `da`, `fi`, `uz`, `pl`, `sv` - [#297](https://github.com/ilyhalight/voice-over-translation/pull/297))
> 
>     * Улучшено форматирование ссылок на YouTube, теперь, язык должен определяться точнее ([#297](https://github.com/ilyhalight/voice-over-translation/pull/297))
> 
>     * Добавлена поддержка нового встраиваемого плеера Bilibili (player.bilibili.com)
> 
>     * vot-cli был перенесен в отдельный **[репозиторий](https://github.com/FOSWLY/vot-cli)**
> 
>     * Удалена папка с web-server (уже давно не используется)
> 
> 
> ## What's Changed (autogenerated)
> 
>     * Улучшения форматирования текста для определения языка by @SashaXser in [Улучшения форматирования текста для определения языка #297](https://github.com/ilyhalight/voice-over-translation/pull/297)
> 
> 
> **Full Changelog**: [1.4.1-beta6...1.4.1-beta7](https://github.com/ilyhalight/voice-over-translation/compare/1.4.1-beta6...1.4.1-beta7)
> _This discussion was created from the release [1.4.1-beta7](https://github.com/ilyhalight/voice-over-translation/releases/tag/1.4.1-beta7)._

Если за ближайшее время не будут найдены баги, то перейдет в релизную ветку

**Reply by NullVerdict** [reply]
Бывает так, что сам YouTube ошибается с субтитрами и может дать английские субтитры в видео на русский язык…
Вот [пример](https://www.youtube.com/watch?v=6n-LLEw6VUQ)

**Reply by ilyhalight** [reply]
> Бывает так, что сам YouTube ошибается с субтитрами и может дать английские субтитры в видео на русский язык…
Вот [пример](https://www.youtube.com/watch?v=6n-LLEw6VUQ)
 
Есть какие-то идеи как это можно было бы исправить? У меня на ум приходит, только, выключить проверку по субтитрам и оставить проверку через описания, заголовок и имя канала, но тогда это будет занимать немного больше времени + язык описания невсегда соответствует языку в видео

**Reply by NullVerdict** [reply]
> > Бывает так, что сам YouTube ошибается с субтитрами и может дать английские субтитры в видео на русский язык…
> > Вот [пример](https://www.youtube.com/watch?v=6n-LLEw6VUQ)
> 
> Есть какие-то идеи как это можно было бы исправить? У меня на ум приходит, только, выключить проверку по субтитрам и оставить проверку через описания, заголовок и имя канала, но тогда это будет занимать немного больше времени + язык описания невсегда соответствует языку в видео

Вариантов особо нет, если только самому сервер транскрибирования не сделать.
